src/property-filter/styles.scss

```diff
@@ -83,6 +85,91 @@
   }
 }
 
+.token-editor-grouped {
+  display: flex;
+  flex-direction: column;
+  gap: awsui.$space-s;
+  user-select: text;
+  margin-block: awsui.$space-xxs;
+  margin-inline: awsui.$space-xxs;
+
+  &-form {
+    margin-block-end: awsui.$space-scaled-l;
+  }
+
+  &-field-property {
+    flex-grow: 2;
+  }
+
+  &-field-operator {
+    flex-grow: 1;
+  }
+
+  &-field-value {
+    flex-grow: 2;
+  }
+
+  &-cancel {
+    margin-inline-end: awsui.$space-xs;
+  }
+
+  &-submit {
+    /* used in test-utils */
```

Comment:
I think this class should be moved to `test-classes` folder

Reply:
The current plan is to replace the existing token editor with the grouped one. In that case we would remove the token-editor-grouped styles and update the existing ones instead and so that the existing &-submit would stay unchanged, lowering the risk of test classes mismatch.